Sources/Shared/Extensions/Dictionary+Tailor.swift

```diff
@@ -7,6 +7,125 @@
 
 public extension Dictionary {
 
+  /// Map value using key into Double
+  ///
+  /// - Parameter name: The name of the key.
+  /// - Returns: An optional Double
+  func double(_ name: String) -> Double? {
+    if let value: Float = property(name) {
```

Comment:
most of the time, I see we can just cast to `NSNumber` and call `number.doubleValue`